recipes/dvv/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set org = "shakeflow" %}
+{% set name = "dvv" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e9387f247aa7c3b50c32a60bd1f605791f8e46d19d182017e7a787170cf44c5f
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    - {{ PYTHON }} -m pip install poetry-core@https://github.com/python-poetry/poetry-core/archive/refs/tags/1.6.1.zip
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+
+
+requirements:
+  build:
+    - python >=3.9                     
+    - poetry-core
+    - setuptools 
+    - numpy
+    - pip  
+
+  host:
+    - python >=3.9    
+    - pip
+    - numpy >=1.25.0
+
+  run:
+    - python >=3.9    
+    - numpy >=1.25.0
+
+test:
+  imports:
+    - dvv
+  commands:
+    - pip check
+    - python -c "import dvv; ver = dvv.__version__; assert ver != '0.0.0'"
+  requires:
+    - pip
+
+about:
+  home: https://github.com/{{ org }}/{{ name }}
```

Comment:
Please detemplate all of your static variables such as name, org so that the URLs are easy to copy into a browser. Same for all packages.